src/sa.h

```diff
@@ -0,0 +1,112 @@
+#include <stdio.h>
+#include <string.h>
+#include <malloc.h>
+#include "parsers/simple-fasta-parser.h"
+#include "parsers/simple-fastq-parser.h"
+#include <stdint-gcc.h>
```

Comment:
Why do you need compiler-specific headers? What I if only have `clang` to compile with? Do you _really_ need this?

Reply:
Ups, I think this is dead code.

Reply:
Yes, it is indeed not used anymore. I believe it was used to import uint8_t, uint16_t, etc.
If we were to use those types, would it be cross-compiler compatible to get them from stdint.h?

Reply:
Yes, that is where the standard says they are